### ISSUE TITLE
Aircraft, PhysicalSize mismatch

### ISSUE THREAD

**Issue by claudejdev** [PygmalionOfCyprus/cmo-db-requests#4299]
### DB Selector

DB3K

### Affected DBID(s)

Aircraft 31416

### Summary of Changes

User [morphin](https://www.matrixgames.com/forums/memberlist.php?mode=viewprofile&u=89634), [spotted an issue with an UAV size](https://www.matrixgames.com/forums/viewtopic.php?f=10201&t=401580). Looking a bit deeper into the DB, there are indeed a few discrepancies.

The following platforms have a RunwayLength value that doesn't match their Length.

----

### PhysicalSize Should be:

- [ ] _Very Large Aircraft (26.1-75m)_ (2004)

_ID dump:_
`818,919,2101,2103,2559,2560,2566,2567,2591,2593,2594,2597,2598,2913,2927,2956,3385,3386,3387,3463,3693,3694,3865,3866,4009,5499,5500,5737,5797,5887,6091,6916,7020`

<details><summary markdown="span">Verbose List</summary>

|ID|Name|Length|
|--|----|------|
|818|S 102B Korpen [Gulfstream G-IV SP]|26.9|
|919|EMB-145H AEWC|29.9|
|2101|EMB-145 AEWC [R-99A]|29.9|
|2103|EMB-145 RS [R-99B]|29.9|
|2559|Tu-154B Careless|48.0|
|2560|Tu-154M Careless|48.0|
|2566|Yak-42 Clobber|36.4|
|2567|Yak-42D Clobber|36.4|
|2591|ATR-72-500|27.2|
|2593|ATR-72-200|27.2|
|2594|ATR-72-210|27.2|
|2597|ATR-72-ASW [Meltem III]|27.2|
|2598|ATR-72 ASW [P-72A]|27.2|
|2913|Saab 2000 AEW&C [Erieye]|27.3|
|2927|EMB-145I AEWC|29.9|
|2956|Gulfstream G-IV SP|26.9|
|3385|YS-11|26.3|
|3386|YS-11EA|26.3|
|3387|YS-11EB|26.3|
|3463|Saab 2000 AEW&C [Erieye]|27.3|
|3693|Tu-154M Careless [Type I ELINT]|48.0|
|3694|Tu-154M Careless [Type II ELINT / SLAR]|48.0|
|3865|EMB-145 AEWC [R-99A]|29.9|
|3866|EMB-145 RS [R-99B]|29.9|
|4009|Dash-8-400|32.81|
|5499|ATR-72-600|27.2|
|5500|ATR-72-600|27.2|
|5737|ATR-72 ASW [RAS-72-500 Sea Eagle]|27.2|
|5797|EMB-145 RS [R-99B]|29.9|
|5887|EMB-145 AEWC [E-99M]|29.9|
|6091|ATR-72MP|27.2|
|6916|Tu-154B Careless|48.0|
|7020|Tu-154B Careless|48.0|
</details>

- [ ] _Large Aircraft (18.1-26m)_ (2003)

_ID dump:_
`45,595,649,798,876,877,1077,1969,2120,2162,2166,2294,2400,2648,3212,3295,3336,3883,3898,3899,3994,3995,4024,4025,4110,4111,4272,4273,4274,4278,4279,4456,4459,4460,4463,4548,4829,4830,5319,5335,5420,5663,6435,6436,6437,6438,6439,6457,6458,6472,6846`

<details><summary markdown="span">Verbose List</summary>

|ID|Name|Length|
|--|----|------|
|45|Merlin HM.1|19.5|
|595|Merlin HC.3|19.5|
|649|AW.101 Merlin Mk.110 ASW [SH-101A]|19.5|
|798|AW.101 Merlin Mk.512 TTT|19.5|
|876|AW.101 Merlin Mk.112 HEW [EH-101A]|19.5|
|877|AW.101 Merlin Mk.413 ASH [MH-101A]|19.5|
|1077|Merlin HM.1|19.4|
|1969|AW.101 Merlin Mk.110 ASW [SH-101A]|19.5|
|2120|Yak-28PP Brewer E|20.34|
|2162|AW.101 Merlin Mk.110 ASW [SH-101A]|19.5|
|2166|Yak-28R Brewer D|20.34|
|2294|Yak-28PP Brewer E|20.34|
|2400|AW.101 Merlin Mk.410 TTH [UH-101A]|19.5|
|2648|Yak-28PP Brewer E|20.34|
|3212|Yak-141 Freestyle|18.36|
|3295|Yak-141 Freestyle|18.36|
|3336|Merlin HM.2|19.4|
|3883|TIF-25K Aerostat|23.16|
|3898|AW.101 Merlin Mk.518 AMCM [MCH-101]|19.5|
|3899|AW.101 Merlin Mk.518 [CH-101]|19.5|
|3994|EMB-120 Brasilia|20.0|
|3995|EMB-120ER Brasilia Advanced|20.07|
|4024|An-140|22.6|
|4025|An-140 [IrAn-140]|22.6|
|4110|Merlin ASaC.5 [Crowsnest]|19.4|
|4111|Merlin ASaC.1 [Interim]|19.4|
|4272|Merlin HC.3|19.5|
|4273|Merlin HC.4|19.5|
|4274|Merlin HC.3A|19.5|
|4278|Merlin HM.1|19.4|
|4279|Merlin HM.1|19.4|
|4456|Bombardier Challenger 601-3A|20.85|
|4459|Bombardier Challenger 600|20.85|
|4460|Bombardier Challenger 601-1A|20.85|
|4463|AW.101 Merlin Mk.611 CSAR [HH-101A Ceasar]|19.5|
|4548|Yak-141 Freestyle|18.36|
|4829|EH-101 Merlin SAR|19.5|
|4830|EH-101 Merlin CSAR|19.5|
|5319|AW.101 Merlin Mk.512 SAR|19.5|
|5335|AW.101 Merlin Mk.110 ASW|19.5|
|5420|KJ-600|18.4|
|5663|CE-144 Challenger|20.85|
|6435|IrAn-140T|22.6|
|6436|IrAn-140AEW|22.6|
|6437|IrAn-140MP [HMS-200]|22.6|
|6438|IrAn-140MP [HMS-400]|22.6|
|6439|IrAn-140MP [HMS-500]|22.6|
|6457|AW.101 Merlin Mk.512 TTT|19.5|
|6458|AW.101 Merlin Mk.512 SAR|19.5|
|6472|AW.101 Merlin Mk.512 SAR Queen|19.5|
|6846|Yak-28P Brewer|20.34|
</details>

- [ ] _Medium Aircraft (12.1-18m)_ (2002)

_ID dump:_
`1300,1301,2127,2493,2558,3151,3181,3182,3183,3195,3324,3325,3341,3450,3451,3452,3453,3454,3484,3485,3882,3916,3944,3945,3947,4032,4085,4098,4129,4135,4140,4151,4162,4177,4190,4191,4192,4193,4194,4238,4254,4255,4256,4257,4258,4259,4342,4343,4350,4435,4436,4437,4577,4916,4917,4932,4955,4998,5009,5017,5019,5062,5103,5128,5129,5178,5209,5210,5211,5212,5240,5241,5277,5296,5297,5339,5340,5343,5344,5345,5348,5362,5364,5365,5439,5440,5442,5501,5519,5520,5548,5571,5626,5633,5636,5779,5793,5856,5861,5863,5944,5967,6014,6040,6097,6106,6155,6156,6207,6223,6240,6278,6284,6299,6357,6358,6359,6363,6377,6442,6588,6641,6687,6690,6697,6741,6804,6905,6950,7001,7031,7033,7070`

<details><summary markdown="span">Verbose List</summary>

|ID|Name|Length|
|--|----|------|
|1300|BN-2 Defender 4000|12.2|
|1301|BN-2T Turbine Islander|12.2|
|2127|Super King Air 200T|13.34|
|2493|Z-10 Fierce Thunderbolt|14.15|
|2558|Super King Air B200|13.34|
|3151|BN-2T Turbine Islander|12.2|
|3181|T-50 Golden Eagle|13.1|
|3182|TA-50 Golden Eagle|13.1|
|3183|FA-50 Golden Eagle|13.1|
|3195|M28B Bryza 1R [An-28 Copy]|13.1|
|3324|Pilatus PC-12|14.4|
|3325|U-28A [Pilatus PC-12]|14.4|
|3341|PZL M28B Bryza 1RM bis [An-28 Copy]|13.1|
|3450|DHC-6-100 Twin Otter|15.77|
|3451|DHC-6-200 Twin Otter|15.77|
|3452|DHC-6-300 Twin Otter|15.77|
|3453|DHC-6-400 Twin Otter|15.77|
|3454|DHC-6-100 Twin Otter|15.77|
|3484|A-12A Avenger II|13.2|
|3485|A-12A Avenger II|13.2|
|3882|Super King Air 200T|13.34|
|3916|Y-5 Colt [An-2]|12.4|
|3944|Cessna AC-208B Combat Caravan|12.67|
|3945|Cessna 208B Grand Caravan|12.67|
|3947|Cessna AC-208B Combat Caravan|12.67|
|4032|Z-10 Fierce Thunderbolt|14.15|
|4085|T-50i Golden Eagle|13.1|
|4098|AW.139|16.66|
|4129|A-12A Avenger II|13.2|
|4135|A-12A Avenger II|13.2|
|4140|A-12A Avenger II|13.2|
|4151|A-12A Avenger II|13.2|
|4162|A-12A Avenger II|13.2|
|4177|A-12A Avenger II|13.2|
|4190|F-16XL Falcon|16.51|
|4191|F-16XL Falcon|16.51|
|4192|F-16XL Falcon|16.51|
|4193|F-16XL Falcon|16.51|
|4194|F-16XL Falcon|16.51|
|4238|FA-50PH Golden Eagle|13.1|
|4254|Cessna 500 Citation I|13.26|
|4255|Cessna 550 Citation II|14.39|
|4256|Cessna 560 Citation V|14.91|
|4257|Cessna 560 Citation Ultra|14.91|
|4258|OT-47B Tracker [SR-560, Cessna 560 Citation Ultra]|14.91|
|4259|L-15 Falcon [JL-10]|12.27|
|4342|Y-12 (II)|14.86|
|4343|Y-12 (II)|14.86|
|4350|Y-12 (II)|14.86|
|4435|IAI 1124 Westwind I|15.93|
|4436|IAI 1124A Westwind II|15.93|
|4437|IAI 1124N Sea Scan [Shahaf]|15.93|
|4577|A-12A Avenger II|13.2|
|4916|MQ-25A Stingray UCAV [Armed]|15.5|
|4917|MQ-25A Stingray UAV|15.54|
|4932|L-15Z Falcon|12.27|
|4955|FA-50 Golden Eagle|13.1|
|4998|Bayraktar Akinci UCAV|12.5|
|5009|Y-12 (II)|14.86|
|5017|AW.149|17.57|
|5019|MQ-20 Avenger UAV|13.4|
|5062|Y-12 (II)|14.86|
|5103|AW.159 Wildcat Mk210|13.5|
|5128|AS.355F2 Ecureuil 2|12.94|
|5129|AS.355F2 Ecureuil 2|12.94|
|5178|T-50TH Golden Eagle|13.1|
|5209|European MALE RPAS|16.0|
|5210|European MALE RPAS|16.0|
|5211|European MALE RPAS|16.0|
|5212|European MALE RPAS|16.0|
|5240|Super King Air B200|13.34|
|5241|DHC-6-320 Twin Otter|15.77|
|5277|MH-139A Grey Wolf|16.7|
|5296|AW.119Kx|12.92|
|5297|TH-73A Thrasher [AW.119Kx]|12.92|
|5339|T-50IQ Golden Eagle|13.1|
|5340|T-50TH Golden Eagle|13.1|
|5343|Super King Air 350|14.22|
|5344|Super King Air 350ER|14.22|
|5345|Super King Air 350|14.22|
|5348|Cessna 208B Grand Caravan|12.67|
|5362|Cessna AC-208B Combat Caravan|12.67|
|5364|Cessna 208B Grand Caravan|12.67|
|5365|Pilatus PC-12NG|14.4|
|5439|Super King Air 350i|14.22|
|5440|Super King Air 360|14.22|
|5442|Super King Air 360|14.22|
|5501|Y-12 (II)|14.86|
|5519|AW.119Ke|12.92|
|5520|AW.119Ke|12.92|
|5548|AW.139|16.66|
|5571|AW.139|16.66|
|5626|C-145A Combat Coyote [An-28 Copy]|13.1|
|5633|L-15 Falcon|12.27|
|5636|KAI Light Armed Helicopter|14.6|
|5779|Turbo Commander 980C|13.1|
|5793|TA-50 Golden Eagle|13.1|
|5856|Z-10 Fierce Thunderbolt|14.15|
|5861|AW.149|17.57|
|5863|AW.139|16.66|
|5944|FA-50GF Golden Eagle|13.1|
|5967|Super King Air 350|14.22|
|6014|Super King Air 350i|14.22|
|6040|FA-50PL Golden Eagle|13.1|
|6097|A-12A Avenger II|13.2|
|6106|AW.139|16.66|
|6155|AW.139|16.66|
|6156|AW.139|16.66|
|6207|AW.139|16.66|
|6223|Y-12 (II)|14.86|
|6240|Turbo Commander 980C|13.1|
|6278|Cessna 208B Grand Caravan|12.67|
|6284|AW.139|16.66|
|6299|Cessna 208B Grand Caravan|12.67|
|6357|AW.139|16.66|
|6358|DHC-6-400 Twin Otter|15.77|
|6359|DHC-6-400 Twin Otter|15.77|
|6363|Cessna 208B Grand Caravan|12.67|
|6377|Y-12 (II)|14.86|
|6442|FA-50 Golden Eagle|13.1|
|6588|DHC-6-400 Twin Otter|15.77|
|6641|FA-50PL Golden Eagle|13.1|
|6687|Pilatus PC-12NG|14.4|
|6690|AW.119Ke Koala|12.92|
|6697|AW.139|16.66|
|6741|AW.149|17.57|
|6804|Bayraktar Akinci UCAV|12.5|
|6905|FA-50PH Golden Eagle|13.1|
|6950|Bayraktar Akinci UCAV|12.5|
|7001|Bayraktar Akinci UCAV|12.5|
|7031|Cessna CJ3 SMA|14.91|
|7033|Z-10ME Fierce Thunderbolt|14.15|
|7070|AW.119Ke Koala|12.92|

</details>

- [ ] _Small Aircraft (0-12m)_ (2001)

_ID dump:_
`1522,1654,2336,2897,4316,4328,5957,6177,6579,6722,6723,6742,6816,6829,6853,6854,6855,6911,6947`

<details><summary markdown="span">Verbose List</summary>

|ID|Name|Length|
|--|----|------|
|1522|Cessna F406 Caravan II [Vigilant]|11.9|
|1654|J-21 Jastreb|10.9|
|2336|J-21 Jastreb|10.9|
|2897|Cessna F406 Caravan II|11.9|
|4316|OH-1 Ninja|12.0|
|4328|RQ-180 UAV|10.0|
|5957|Cessna F406 Caravan II|11.9|
|6177|Shafaq|10.84|
|6579|Mi-2B Hoplite|11.9|
|6722|V-247 Vigilant UAV|11.0|
|6723|V-247 Vigilant UAV|11.0|
|6742|NJ-21 Jastreb|10.9|
|6816|Shafaq|10.84|
|6829|Mi-2B Hoplite|11.9|
|6853|Mi-2A Hoplite|11.9|
|6854|Mi-2MSB-5 Hoplite|11.9|
|6855|Mi-2A Hoplite|11.9|
|6911|Mi-2A Hoplite|11.9|
|6947|Mi-2A Hoplite|11.9|
</details>

### Sources

DB3K_504

#### Note

I left out a few platforms as I've noticed they need a separate update request.
A notable exception is the SH-2 Seasprite. With its 12.2 meters length, it should technically be considered as a Medium A/C. However, I suspect that such a change might: a) make a mess with several scenarios, and b) might be realistic since our data points might not be granular enough to simulate the many ways equipment was made to fit in their place in actual operations.
TLDR; for naval helo, I don't think 0.1m is worth the trouble.

**Comment by claudejdev** [comment]
Is such a report still useful now that you guys did get the RAMDB DLC?
